src/concrete/ml/pytest/utils.py

```diff
@@ -388,6 +388,11 @@ def instantiate_model_generic(model_class, n_bits, **parameters):
         model_name (str): The type of the model as a string.
         model (object): The model instance.
     """
+    # Force multi threaded models to be single threaded as it is not working
```

Comment:
Maybe add an issue here

Reply:
Not sure what kind of follow up issue we could make. xgboost,  pytest and new simulation are all trying to access all threads so we have to let one of them on all threads and set the rest single thread.